error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/BigDecimalRulesTestInput.java

```diff
@@ -18,6 +18,6 @@ ImmutableSet<BigDecimal> testBigDecimalTen() {
   }
 
   ImmutableSet<BigDecimal> testBigDecimalFactoryMethod() {
-    return ImmutableSet.of(new BigDecimal(0), new BigDecimal(0L));
+    return ImmutableSet.of(new BigDecimal(2), new BigDecimal(2L), new BigDecimal(2.0));
```

Comment:
felt `2` was a nicer value to use here as there is a rule above to rewrite `0` to `ZERO` as well